src/gui/settings/foundObjects.html

```diff
@@ -302,6 +353,40 @@
         }
     }
 
+    function uploadTarget(node) {
+      var button = node.parentElement.querySelector('.uploadTargetButton');
+      button.innerText = "Uploading...";
+
+      // Set up the request.
+      var xhr = new XMLHttpRequest();
+
+      var postUrl = 'http://' + node.dataset.ip + ':' + node.dataset.port + '/content/' + node.dataset.name; // set to target upload endpoint on server
+
+      // Open the connection.
+      xhr.open('POST', postUrl, true);
+
+      // Set up a handler for when the request finishes.
+      xhr.onload = function () {
+          if (xhr.status === 200) {
+              // File(s) uploaded.
+              console.log('successful upload');
+              button.innerText = "Success!";
+              document.querySelector('.reload-notice').removeAttribute('hidden');
+              setTimeout(function() {
+                button.innerText = "Upload Target";
+              }, 3000);
+          } else {
+              console.log('error uploading');
```

Comment:
It would be good for this to bubble up the same way as the success message, ideally with an explanation of the error